controllers/controllers/services/osbapi/client.go

```diff
@@ -298,6 +298,33 @@ func (c *Client) Unbind(ctx context.Context, payload UnbindPayload) (UnbindRespo
 	return response, nil
 }
 
+func (c *Client) GetServiceBinding(ctx context.Context, payload BindPayload) (BindingResponse, error) {
+	statusCode, respBytes, err := c.newBrokerRequester().
+		forBroker(c.broker).
+		sendRequest(
+			ctx,
+			"/v2/service_instances/"+payload.InstanceID+"/service_bindings/"+payload.BindingID,
+			http.MethodGet,
+			nil,
```

Comment:
According to the [OSBAPI spec](https://github.com/openservicebrokerapi/servicebroker/blob/master/spec.md#parameters-7), broker clients could provide the plan and offering ids as query parameters (as brokers seem to find such hints useful).

We are providing those IDs in the rest of the methods in the clients, we should do that here as well.